.travis.yml

```diff
@@ -7,7 +7,6 @@ node_js:
   - "4"
   - "6"
   - "8"
-  - "10"
```

Comment:
Try replace the default task in Gruntfile and run grunt again:

```
grunt.registerTask('default', ['test', 'contrib-ci:skipIfExists', 'contrib-core']);
```

I have already made a PR upstream to add node.js 10 so after that this won't be needed